rosdep/base.yaml

```diff
@@ -2979,6 +2994,7 @@ libssh2-dev:
   gentoo: [net-libs/libssh2]
   ubuntu: [libssh2-1-dev]
 libssl-dev:
+  alpine: [openssl1.0-dev]
```

Comment:
Alpine looks recommending to use libressl instead of openssl (https://bugs.alpinelinux.org/issues/4970).
Using openssl may cause conflict with libressl.